openqa-mail-notification/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "2.1"
+services:
+  openqa-mail-notification:
+    build:
+      dockerfile: Dockerfile
+      context: .
```

Comment:
```suggestion
      context: ./build_context
```

Reply:
This will make the docker composer to look for the `Dockerfile` inside the `build_context` directory... Is this what you had in mind?

Reply:
But, this is just a suggestion based on my taste. You can also just rely on a .dockerignore on the root of the github project.

Reply:
Ok, then I'll drop this suggestion if you don't mind. Thanks though!